packages/ledger/src/lib/ledger.spec.ts

```diff
@@ -1,37 +1,42 @@
 /* eslint-disable @nrwl/nx/enforce-module-boundaries */
 import { mock } from "jest-mock-extended";
 
-import { mockWallet } from "../../../core/src/lib/testUtils";
-import type { MockWalletDependencies } from "../../../core/src/lib/testUtils";
-import type { HardwareWallet } from "../../../core/src/lib/wallet";
-import type {
+import {
+  mockWallet,
+  MockWalletDependencies,
+} from "../../../core/src/lib/testUtils";
+import { HardwareWallet, Transaction } from "../../../core/src/lib/wallet";
+import {
   ProviderService,
   JsonStorageService,
 } from "../../../core/src/lib/services";
 import { LedgerClient } from "./ledger-client";
+import {
+  ReadOnlyStore,
+  WalletSelectorState,
+} from "packages/core/src/lib/store.types";
```

Comment:
This import should be `../../../core/src/lib/store.types`.
We should also use `import type` when import types as we were previously in this file